### DISCUSSION TITLE
有社区或群聊吗

### DISCUSSION THREAD

**Discussion by hts0000** [ZuodaoTech/everyone-can-use-english#1127] [General]
方便交流经验以及形成社区最佳实践

**Comment by an-lee** [comment]
暂时只有 [GitHub Discussions](https://github.com/ZuodaoTech/everyone-can-use-english/discussions) 和 [Mixin 群组](https://mixin.one/codes/f6ff96b8-54fb-4ad8-a6d4-5a5bdb1df13e)。

有计划开一个专门的社区

**Comment by hh7418695** [comment]
请问，我下载了应用，点击链接为啥说链接无效呀
